README.md

```diff
@@ -141,4 +141,4 @@ Thanks goes to these wonderful people ([emoji key](https://allcontributors.org/d
 This project follows the [all-contributors](https://github.com/all-contributors/all-contributors) specification. Contributions of any kind welcome!
 To see what contributors feel they've done in their own words, please see our [contribution recognition page][contribution].
 
-[contribution]: <contributions.md>
\ No newline at end of file
+[contribution]: <contributions.md>
```

Comment:
Huh, any idea what's going on here?

Reply:
I think the version on `main` does not end with a newline character, whereas vim probably added it when I edited the file.

Reply:
I see. In that case I think your PR has the preferred style, though we haven't been picky about the line breaks in the past.